.gitignore

```diff
@@ -1 +1,5 @@
+# For Vim users
+.nvimrc
+.vimrc
```

Comment:
Note: this is fine to have both in project-s .gitignore, and in your personal in `~/.gitignore` or `~/.config/git/ignore`.

Some people even [advocate to always keep your personal editor's settings in your personal files](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/). I think this is a cleaner practice indeed